NEWgIT.Core/AnalysisDTO.cs

```diff
@@ -0,0 +1,11 @@
+namespace NEWgIT.Core;
```

Comment:
Er det meningen at alle variabel navnene starter med småt udover "Id"?

Reply:
Det er gennem hele DTO filen. Kunne ikke lige finde ud af at kommentere på en hel klasse i stedet for en linje